Packs/ThalesCipherTrustManager/Integrations/ThalesCipherTrustManager/ThalesCipherTrustManager.yml

```diff
@@ -0,0 +1,2389 @@
+      predefined:
+        - '256'
+        - '384'
+        - '521'
+        - '1024'
+        - '2048'
+        - '3072'
+        - '4096'
+      auto: PREDEFINED
+    outputs:
+    - contextPath: InfoFile.Name
+      description: FileName
+      type: string
+    - contextPath: InfoFile.EntryID
+      description: The EntryID of the report
+      type: string
+    - contextPath: InfoFile.Size
+      description: File Size
+      type: number
+    - contextPath: InfoFile.Type
+      description: File type e.g. "PE"
+      type: string
+    - contextPath: InfoFile.Info
+      description: Basic information of the file
+      type: string
+    - contextPath: InfoFile.Extension
+      description: File extension
+      type: string
+    - contextPath: CipherTrust.LocalCA.id
+      description: Unique identifier for the CA.
+      type: String
+    - contextPath: CipherTrust.LocalCA.uri
+      description: Uniform Resource Identifier for the CA.
+      type: String
+    - contextPath: CipherTrust.LocalCA.account
+      description: Account associated with the CA.
+      type: String
+    - contextPath: CipherTrust.LocalCA.application
+      description: Application associated with the CA.
+      type: String
+    - contextPath: CipherTrust.LocalCA.devAccount
+      description: Developer account associated with the CA.
+      type: String
+    - contextPath: CipherTrust.LocalCA.createdAt
+      description: Timestamp when the CA was created.
+      type: Date
+    - contextPath: CipherTrust.LocalCA.updatedAt
+      description: Timestamp when the CA was last updated.
+      type: Date
+    - contextPath: CipherTrust.LocalCA.name
+      description: Name of the CA.
+      type: String
+    - contextPath: CipherTrust.LocalCA.state
+      description: State of the CA.
+      type: String
+    - contextPath: CipherTrust.LocalCA.subject
+      description: Distinguished Name (DN) of the CA subject.
+      type: String
+    - contextPath: CipherTrust.LocalCA.notBefore
+      description: Timestamp before which the certificate is not valid.
+      type: Date
+    - contextPath: CipherTrust.LocalCA.notAfter
+      description: Timestamp after which the certificate is not valid.
+      type: Date
+    - contextPath: CipherTrust.LocalCA.sha1Fingerprint
+      description: SHA-1 fingerprint of the CA certificate.
+      type: String
+    - contextPath: CipherTrust.LocalCA.sha256Fingerprint
+      description: SHA-256 fingerprint of the CA certificate.
+      type: String
+    - contextPath: CipherTrust.LocalCA.sha512Fingerprint
+      description: SHA-512 fingerprint of the CA certificate.
+      type: String
+  - deprecated: false
+    description: Deletes a local CA certificate.
+    name: ciphertrust-local-ca-delete
+    arguments:
+    - name: local_ca_id
+      isArray: false
+      description: An identifier of the resource. This can be either the ID (a UUIDv4),the Name, the URI, or the slug (which is the last component of the URI).
```

Comment:
```suggestion
      description: An identifier of the resource. This can be either the ID (a UUIDv4), the name, the URI, or the slug (which is the last component of the URI).
```